doc/manipulating_images/data_preparation.rst

```diff
@@ -70,36 +70,35 @@ relevant for the research questions at hand.
 simplifying these "data folding" steps that often preceed the actual
 statistical analysis.
 
-On an advanced note,
-the underlying philosophy of these classes is similar to `scikit-learn
-<http://scikit-learn.org>`_\ 's
-transformers. First, objects are initialized with some parameters guiding
-the transformation (unrelated to the data). Then the fit() method
+On an advanced note, the underlying philosophy of these classes is similar
+to `scikit-learn <http://scikit-learn.org>`_\ 's transformers.
+First, objects are initialized with some parameters guiding
+the transformation (unrelated to the data). Then the `fit()` method
 should be called, possibly specifying some data-related
 information (such as number of images to process), to perform some
 initial computation (e.g., fitting a mask based on the data). Finally,
-transform() can be called, with the data as argument, to perform some
-computation on data themselves (e.g. extracting time series from images).
+`transform()` can be called, with the data as argument, to perform some
+computation on data themselves (e.g., extracting time series from images).
 
 Note that the masker objects may not cover all the image transformations
-for specific tasks. Users who want to make some specific processing may
-have to call low-level functions (see e.g. :mod:`nilearn.signal`,
-:mod:`nilearn.masking`).
+for specific tasks. Users who want to make some specific processing
+may have to call low-level functions
+(see e.g., :mod:`nilearn.signal`, :mod:`nilearn.masking`).
```

Comment:
> There is no formal rule, I agree, except consistency

My 2 cents: consistency about this kind of things is very hard if not impossible to maintain in the long run and the benefit/cost ratio of maintaining it is very low.